Mathlib/RingTheory/Artinian.lean

```diff
@@ -322,10 +322,10 @@ theorem isArtinian_of_submodule_of_artinian (R M) [Ring R] [AddCommGroup M] [Mod
     (N : Submodule R M) (_ : IsArtinian R M) : IsArtinian R N := inferInstance
 #align is_artinian_of_submodule_of_artinian isArtinian_of_submodule_of_artinian
 
-theorem isArtinian_of_quotient_of_artinian (R) [Ring R] (M) [AddCommGroup M] [Module R M]
-    (N : Submodule R M) (_ : IsArtinian R M) : IsArtinian R (M ⧸ N) :=
+instance isArtinian_of_quotient_of_artinian (R) [Ring R] (M) [AddCommGroup M] [Module R M]
+    (N : Submodule R M) [IsArtinian R M] : IsArtinian R (M ⧸ N) :=
   isArtinian_of_surjective M (Submodule.mkQ N) (Submodule.Quotient.mk_surjective N)
-#align is_artinian_of_quotient_of_artinian isArtinian_of_quotient_of_artinian
+#noalign is_artinian_of_quotient_of_artinian
```

Comment:
Even though one explicit argument changes to instance implicit, I guess this should still be perfectly aligned?
```suggestion
#align is_artinian_of_quotient_of_artinian isArtinian_of_quotient_of_artinian
```